### PR TITLE
Randomize names for classification integration tests

### PR DESCRIPTION
**Pull request by marko-polo-cheno**

### CODE REVIEW

**Comment by marko-polo-cheno** [comment]
These tests will become obsolete once prebuilt-classification-workflow tests appear

File: tests/integration/classification/multiclass/test_workflow.py [hunk 1]
```diff
@@ -31,11 +28,26 @@
...
 
 
-@pytest.mark.parametrize("inference_type", [ScoredClassificationLabel, InferenceLabel])
-def test__multiclass_classification__smoke(
-    inference_type: Union[Type[ScoredClassificationLabel], Type[InferenceLabel]],
-) -> None:
-    name = with_test_prefix(f"{__file__} test__multiclass_classification__smoke ({inference_type.__name__})")
+def test__multiclass_classification__smoke__scored__classification_label() -> None:
+    name = with_test_prefix(
+        f"{__file__} test__multiclass_classification__smoke (ScoredClassificationLabel) {str(uuid.uuid4())}",
```
Comment (ly-kolena) [inline tests/integration/classification/multiclass/test_workflow.py:33]: `with_test_prefix` includes a session-wide random string already, is the additional uuid needed for correctness?

File: tests/integration/classification/multiclass/test_workflow.py [hunk 2]
```diff
@@ -31,11 +28,26 @@
...
+    def infer(_: TestSample) -> Inference:
+        score = random.random()
+        inferences = [("example", 1 - score), ("another", score / 2), ("third", score / 2)]
+        return Inference(inferences=[ScoredClassificationLabel(label, conf) for label, conf in inferences])
+
+    model = Model(f"{name} model", infer=infer)
+    test(model, test_suite)
+
+
+def test__multiclass_classification__smoke__inference__label() -> None:
```
Comment (ly-kolena) [inline tests/integration/classification/multiclass/test_workflow.py:49]: What is the new test verifying? It looks like the existing test, just with different label class, yet we don't have verification based on label class.

Comment (ly-kolena) [reply tests/integration/classification/multiclass/test_workflow.py:49]: I understand now, seeing the `parametrize` annotation.

**Review by ly-kolena — APPROVED** [review]

**Review by gordonhart — APPROVED** [review]
Do you know why this only now started failing?

**Comment by marko-polo-cheno** [comment]
@ly-kolena @gordonhart [Rerunning](https://github.com/kolenaIO/kolena/runs/14801447876) shows no issues. I assumed the issue might have been some secret cacheing between the `parametrize` params. Maybe it was just a network hiccup?

File: tests/integration/classification/multiclass/test_workflow.py (hunk 1 above)
Comment (marko-polo-cheno) [reply tests/integration/classification/multiclass/test_workflow.py:33]: In this case, no - just an extra layer to ensure the test cases are unique.

**Comment by gordonhart** [comment]
Looks like the [test failed due to a 502 Bad Gateway](https://app.circleci.com/pipelines/github/kolenaIO/kolena/1033/workflows/20b109c0-7226-41f9-8bcd-078653181a65/jobs/16559?invite=true#step-103-298), would make sense that it succeeded on retrigger. Did you see any logs indicating that this fixture was the culprit?

**Comment by marko-polo-cheno** [comment]
> Did you see any logs indicating that this fixture was the culprit?

Couldn't find much information for the source of error, but going to close this PR seeing that the original tests are passing.

**Pull request closed without merging**